### PR TITLE
implemented text search

### PR DESCRIPTION
**Pull request by joutad**
**Why**

There was a needle in the haystack so I developed a program that spots the needle
---

**What**

Added a directory that provides the solution file
---

**Verification**

others can test and verify this feature by changing into my directory and then using the command `python3 searcher.py`
---

**Checklist:**

- [💯] This contribution abides by all rules listed in `docs/Rules.md`

### CODE REVIEW

File: joudat_dir/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+`How to run the file:`
```
Comment (SoorajModi) [inline joudat_dir/README.md]: This folder should be under `submissions` not under the root directory

File: joudat_dir/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+```python3 searcher.py```
+
+Enjoy!
```
Comment (SoorajModi) [inline joudat_dir/README.md]: To the README you should add:
- The name of the project
- A list of contributors
- What it does
- Why you contributed it

**Review by SoorajModi — CHANGES_REQUESTED** [review]

File: submissions/basic_re/searcher.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import re
+
+str = "there is a needle in the haystack"
+
+x = re.findall(r'needle', str, re.IGNORECASE)[0]
+
+print("FOUND:", x)
```
Comment (MarshallAsch) [inline submissions/basic_re/searcher.py]: Add a trailing newline at the end of the file. see [here](https://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap03.html#tag_03_392) if your interested in learning why

File: joudat_dir/searcher.py
Comment (MarshallAsch) [inline joudat_dir/searcher.py:1]: Remove this file. that has been moved.

**Review by MarshallAsch — CHANGES_REQUESTED** [review]

**Review by MarshallAsch — APPROVED** [review]

**Review by SoorajModi — APPROVED** [review]

**Pull request merged**